### ISSUE TITLE
Input string was not in a correct format. in 4 and 5th version. 

### ISSUE THREAD

**Issue by tsviet** [fluentassertions/fluentassertions#709]
{"mon_1_open":"","mon_1_close":"","tue_1_open":"","tue_1_close":"","wed_1_open":"","wed_1_close":"","thu_1_open":"","thu_1_close":"","fri_1_open":"","fri_1_close":"","sat_1_open":"","sat_1_close":"","sun_1_open":"","sun_1_close":"","GetBusinessHours":{"Monday":{},"Tuesday":{},"Wednesday":{},"Thursday":{},"Friday":{},"Saturday":{},"Sunday":{}}}

HoursText.Should().Contain("mon")

    at System.Text.StringBuilder.FormatError()
   at System.Text.StringBuilder.AppendFormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.Format(String format, Object[] args)
   at FluentAssertions.Execution.AssertionScope.BecauseOf(String because, Object[] becauseArgs)
   at FluentAssertions.Primitives.StringAssertions.Contain(String expected, String because, Object[] becauseArgs)
   at

**Comment by tsviet** [comment]
Latest betta. 5.0.0-beta.4

**Comment by dennisdoomen** [comment]
OK. Thanks for reporting that.

**Comment by jnyrup** [comment]
@tsviet Which framework are you targeting?
I just tried to reproduce this by creating an empty .NET 4.6.1 project only referencing FluentAssertions 5.0.0-beta4 and mstestV2 1.20, and I my test passed?

```csharp
[TestClass]
public class Class1
{
	[TestMethod]
	public void MyTestMethod()
	{
		string actual = "{\"mon_1_open\":\"\",\"mon_1_close\":\"\",\"tue_1_open\":\"\",\"tue_1_close\":\"\",\"wed_1_open\":\"\",\"wed_1_close\":\"\",\"thu_1_open\":\"\",\"thu_1_close\":\"\",\"fri_1_open\":\"\",\"fri_1_close\":\"\",\"sat_1_open\":\"\",\"sat_1_close\":\"\",\"sun_1_open\":\"\",\"sun_1_close\":\"\",\"GetBusinessHours\":{\"Monday\":{},\"Tuesday\":{},\"Wednesday\":{},\"Thursday\":{},\"Friday\":{},\"Saturday\":{},\"Sunday\":{}}}";

		string expected = "mon";

		actual.Should().Contain(expected);
	}
}
```

Am I missing something in my attempt to reproduce?

**Comment by tsviet** [comment]
A test you created working for me fine. I tried to save a string to a file and test works fine too. But when it comes from web request it fails. Looks like I auto-generates this from JSON object. 

![capture](https://user-images.githubusercontent.com/13110596/34267479-e2728646-e631-11e7-9121-c80c93d516a8.PNG)

I tried to reproduce it in isolation but test was success
`[Fact]
		public void MyTest()
		{
			var json = File.ReadAllText("C:/temp/test.txt");
			
			var facebookData = JsonConvert.DeserializeObject<FacebookData>(json);
			
			var prop = new ReviewValidationProperties();
			
			prop.CurrentBusinessHoursText = facebookData?.hours?.SerializeToString();

			prop.CurrentBusinessHoursText.Should().Contain("mon");
               }
}`

That leads that I need to optimize my code and not serialize empty values and craft real values other than that empty JSON. 

Think that issue can be closed.
